devicetypes/fibargroup/fibaro-walli-double-switch.src/fibaro-walli-double-switch.groovy

```diff
@@ -381,10 +345,7 @@ def childOnOff(deviceNetworkId, value) {
 private onOffCmd(value, endpoint = 1) {
 	delayBetween([
 		encap(zwave.basicV1.basicSet(value: value), endpoint),
-		encap(zwave.basicV1.basicGet(), endpoint),
-		"delay 3000",
-		encap(zwave.meterV3.meterGet(scale: 0), endpoint),
-		encap(zwave.meterV3.meterGet(scale: 2), endpoint)
```

Comment:
were these meter gets causing problems? Does the device update us automatically?

Reply:
Device updates automatically, it was noticed that those additional meterGets choke that device sometimes.